lib/errors.js

```diff
@@ -109,7 +108,8 @@ exports.Err = class {
 
         return format.replace(/\{\{(\!?)([^}]+)\}\}/g, ($0, isSecure, name) => {
 
-            const value = Hoek.reach(this.context, name);
+            const context = Object.assign({}, this.context, { key: this.context.label });
```

Comment:
@Marsup I wasn't fixing the mistake with the template language, I was fixing the mistake with the context in `err.details`.